cob_hardware_config/raw3-5/config/base_controller.yaml

```diff
@@ -24,15 +24,15 @@ twist_controller:
       d_phi_max: 10.0 #100.0
       dd_phi_max: 40.0 #100.0
   wheels: &wheels
-    - steer: fl_caster_rotation_joint
-      drive: fl_caster_r_wheel_joint
-      steer_neutral_position: -180.0
     - steer: bl_caster_rotation_joint
       drive: bl_caster_r_wheel_joint
       steer_neutral_position: -180.0
     - steer: br_caster_rotation_joint
       drive: br_caster_r_wheel_joint
       steer_neutral_position: -180.0
+    - steer: fl_caster_rotation_joint
```

Comment:
In our case, the order in this file did not change anything. The order in the base_driver.yaml file influenced our initialization